code/src/nuvla/api/api.py

```diff
@@ -567,7 +565,7 @@ def add(self, resource_type, data):
         :rtype:     CimiResponse
         """
         collection = self.search(resource_type=resource_type, last=0)
-        operation_href = self._cimi_find_operation_href(collection, 'add')
+        operation_href = f'{collection.id}'
         return CimiResponse(self._cimi_post(resource_id=operation_href, json=data))
 
```

Comment:
Why f-string needs to be used? Is `id` on the `collection` not of type str? And also, I don't see a need for the intermediary `operation_href` var.